### ISSUE TITLE
Familiar notes in critical notes

### ISSUE THREAD

**Issue by maras** [maieul/ledmac#695]
Hello. 

First of all, thank you for your work with reledmac package.

Is there any way to insert familiar note in the text of critical note? Something like that
`\edtext{text text text}{\Afootnote{text text\footnoteA{text text} text text}}`
Now I can compile file with such note and there is number of familiar note in the text of critical note, but there is no text of familiar note itself in the block of familiar notes. I didn't find anything about such possibility in manual. Maybe I miss something? It would be nice to have such possibility.
Thank you.

**Comment by maieul** [comment]
you mean with familiar footnote mark in the critical footnote?

For now, it is not possible, and it will be complex to implement. And not in my priority.

**Comment by maras** [comment]
Yes, footnote mark, sorry.
Certain substitute can be \ledsidenote in critical notes, but they don't working too, as I can see.

**Comment by maieul** [comment]
yes, also. But I have no time for these two features (which are, indeed, real strange…)

**Comment by maras** [comment]
There are several highly corrected and revised versions of one document that I want to critically publish. More than two, so I can't use reledpar. I put information about corrections and differences between variants into critical notes (two series). So I have some information only in critical apparatus, that need to be explained not in apparatus itself (because apparatuses are for versions and corrections, and they are already huge). Maybe it was wrong strategy of publishing, but broader explanations should be in familiar notes, not in textological apparatus. That is why I am interesting in such feature.
Critical notes is not working in critical notes too. So, there is no way to comment text of critical note.

**Comment by maieul** [comment]
yes, I know.

But there is also no time for me to implement such feature…

**Comment by maras** [comment]
I understand. Thank you for maintaining package. 
Maybe other packages like manyfoot or bigfoot will do the work. I hope they'll work with reledmac.

**Comment by martensas** [comment]
Is this still not possible?

**Comment by maieul** [comment]
yes, still not anytime for reledmac develleoping.
